compiler/optimizer/PreExistence.cpp

```diff
@@ -47,7 +47,7 @@ TR_PrexArgument::knowledgeLevel(TR_PrexArgument *pa)
 TR_PrexArgument::TR_PrexArgument(
       TR::KnownObjectTable::Index knownObjectIndex,
       TR::Compilation *comp) :
-   _classKind(ClassIsUnknown),
+   _classKind(ClassIsFixed),
```

Comment:
Could you please clarify the rationale for this default changing ?

Reply:
Because `_knownObjectIndex` is set, which indicates the arg is fixed class. But after a second thought, I think it's better to put it in the body as we don't want to have a fixed class with unknown class.